rasa/nlu/featurizers/featurzier.py

```diff
@@ -10,6 +10,21 @@
 )
 
 
+def sequence_to_sentence_embedding(
```

Comment:
not sure about the name `embedding`

Reply:
changed it to `sequence_to_sentence_features`, better?

Reply:
do we plan to use this method further? Currently it converts sparse to dense, but we'd like to make this conversion latter during batching

Reply:
the plan is to keep this method for the `SklearnIntentClassifier`, for example. So, for extractors/classifiers that cannot handle a sequence. We can convert it back to sparse if needed.